src/scenes/AcceleratorRouter/Voting/VotingEdit.tsx

```diff
@@ -47,11 +48,17 @@ const VotingEdit = () => {
     if (votes.some((vote) => vote.voteOption === 'Conditional' && !vote.conditionalInfo)) {
       return;
     }
+
+    const updatedVotes = votes.filter(({ conditionalInfo, userId, voteOption }) => {
+      const orignal = phaseVotes.votes.find((originalVote) => originalVote.userId === userId)!!;
+      return orignal.conditionalInfo === conditionalInfo && orignal.voteOption === voteOption;
```

Comment:
should this be `original.conditionalInfo !== conditionalInfo || original.voteOption !== voteOption`?

Reply:
You are so right I forgot to negate the whole thing